src/hooks/use-products.ts

```diff
@@ -24,7 +29,7 @@ export const Products = () => {
 
     try {
       const res = await axios.get(
-        `https://dummyjson.com/products?limit=${LIMIT}&skip=${pagination.skip}`,
+        `https://dummyjson.com/products${query ? `/search?q=${query}&` : '?'}limit=${LIMIT}&skip=${pagination.skip}${sort ? `&sortBy=${sort.sortBy}&order=${sort.order}` : ''}`,
       );
```

Comment:
[Wheel is already invented ](https://axios-http.com/docs/req_config):) (adjust accordingly)
```
const res = await axios.get(
        'https://dummyjson.com/products',
        {  params: { q: query, limit: LIMIT, skip: pagination.skip, ...sort } }
      );
```

Reply:
I was feeling something weird about this. That’s a great piece of code suggestion, thanks for the snippet! I’ll update the request to use the params object.